ocr/tess/crop_morphology.py

```diff
@@ -210,17 +246,53 @@ def crop_in_border(crop):
         int_area = crop_area(intersect_crops(crop, this_crop))
         new_crop = crop_in_border(union_crops(crop, this_crop))
         if 0 < int_area < this_area and crop != new_crop:
-            print '%s -> %s' % (str(crop), str(new_crop))
+            print('%s -> %s' % (str(crop), str(new_crop)))
             changed = True
             crop = new_crop
 
     if changed:
-        return pad_crop(crop, contours, edges, border_contour, pad_px)
+        return pad_crop(crop, contours, edges, border_contour)
     else:
         return crop
 
 
-def downscale_image(im, max_dim=2048):
+def remove_stamp(edges: Image, path: str):
```

Comment:
This was maybe a little silly, but stamps can sometimes throw off text extraction. Example: 710544b